app/experimenter/experiments/constants.py

```diff
@@ -366,6 +366,14 @@ class EmailType(models.TextChoices):
         Application.FOCUS_IOS: Version.FIREFOX_101,
     }
 
+    ROLLOUT_SUPPORT_VERSION = {
+        Application.DESKTOP: Version.FIREFOX_105,
+        Application.FENIX: Version.FIREFOX_105,
+        Application.FOCUS_ANDROID: Version.FIREFOX_105,
+        Application.IOS: Version.FIREFOX_105,
+        Application.FOCUS_IOS: Version.FIREFOX_105,
```

Comment:
Actually the other good thing mypy can do is force you to be exhaustive about enums (in some cases anyways).  That's also useful.